get-authors.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env sh
+
+REPO_URL=$1
+EMAIL_DOMAIN=$2
+OUTPUT_PATH=${3:-authors.txt}
+
```

Comment:
Looks good, but you could have used an XML processor to make it  tidy.

`xmlstarlet` is a sorta swiss army knife for XML processing

    apt-get install xmlstarlet

Then the below command would habe done the job.

    svn log -l10 --xml | \
    xmlstarlet sel -t -v '//author/text()' --nl - | \
    sort | uniq | while read u; do \
        echo "$u = $(echo $u | tr . ' ' | sed 's/\([a-z]\)\([a-zA-Z0-9]*\)/\u\1\2/g') <$u@redmatter.com>"; \
    done

The formula above is roughly based on your logic; main difference though is to pre-process the name from `svn log` to remove the dot  if there is any, before capitalising the first letter of name components.